docs/zh/start/file/minio.md

```diff
@@ -0,0 +1,48 @@
+# 基于MinIO OSS实现文件的增删改查
+
+## 快速开始
+
+Layotto提供了访问文件的示例 [demo](../../../../demo/file/client.go),该示例实现了文件的增删改查操作。
+
+### 第一步：启动layotto
+
+layotto提供了minio的配置文件[oss配置](../../../../configs/config_file.json)，如下图所示
+
+![img.png](../../../img/file/minio.png)
```

Comment:
By the way,it's OK to use a relative path for image,since it can be found under "mosn.io/layotto"